app-android/src/main/java/com/ustadmobile/port/android/view/PersonEditFragment.kt

```diff
@@ -337,6 +355,37 @@ class PersonEditFragment: UstadEditFragment<PersonWithAccount>(), PersonEditView
             }
         })
 
+        mBinding?.firstnamesText?.addTextChangedListener(object: TextWatcher {
```

Comment:
To avoid repetition here, let's create a small util class that does this, e.g.

```
class ClearErrorTextWatcher(val textinputlayout: TextInputLayout) {
  ...
  override onTextChanged(...) {
     textinputlayout.isErrorEnabled = false
     textinputlayout.error = null
  }
}
```